src/endpoints/http/collections/index.js

```diff
@@ -1,18 +1,65 @@
-import listCollections from './list';
-import {createCollection} from './create';
+import listCollectionsInfo from './list';
+import createCollection from './create';
+import deleteCollections from './delete';
 
 export function collectionsHandler(router) {
   //list collection info
   router.get('/collections', function(req, res, next) {
     var appname = req.param('appname');
     req.log.debug({app: appname}, 'listing collections for app');
-    listCollections(req.param('appname'), req.log, req.db)
+    listCollectionsInfo(req.param('appname'), req.log, req.db)
       .then(result => {
         req.log.trace({app: appname, result}, 'collection data listed');
         res.json(result);
       })
       .catch(next);
   });
+
   //create collection
-  createCollection(router);
-}
+  router.post('/collections', (req, res, next) => {
+    if (!req.body.name) {
+      return res.status(400).send('name is required');
```

Comment:
I know this is my code but can this be replaced with `next({'message': 'name is required', code; 400});` ?